src/libpriv/rpmostree-postprocess.c

```diff
@@ -463,12 +463,20 @@ convert_var_to_tmpfiles_d_recurse (GOutputStream *tmpfiles_out,
           filetype_c = 'L';
           break;
         default:
-          g_print ("Ignoring non-directory/non-symlink '%s/%s'\n",
-                   prefix->str,
-                   dent->d_name);
-          if (!glnx_unlinkat (dfd_iter.fd, dent->d_name, 0, error))
-            return FALSE;
-          continue;
+          /* nfs-utils in RHEL7; https://bugzilla.redhat.com/show_bug.cgi?id=1427537 */
+          if (g_str_has_prefix (prefix->str, "/var/lib/nfs"))
```

Comment:
Let's place this under a `case DT_REG:`? (And then fall through to the default case otherwise).

Reply:
Fixup :arrow_down: